rfcs/0181-list-index-syntax.md

```diff
@@ -0,0 +1,290 @@
+---
+feature: list-index-syntax
+start-date: 2024-07-14
+author: rhendric
+co-authors:
+shepherd-team:
```

Comment:
I've been wanting this for so long, very nice proposal, nominating myself as a shepherd!

```suggestion
shepherd-team: @infinisil
```